README.md

```
@@ -1,3 +1,117 @@
+
+```bash
+cp backend/.env.example backend/.env
+nano backend/.env
+```
+
+Set your database credentials, JWT secret, and (optionally) federation variables:
+
+- `FEDERATION_POSTS_API` — (optional) Remote API for posts (e.g. your Vercel deployment)
+- `FEDERATION_USERINFO_API` — (optional) Remote API for user info
+- `FEDERATION_VIDEOS_API` — (optional) Remote API for videos
+- `FEDERATION_BASE_URL` — The public URL of your backend (e.g. https://yourdomain.com)
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)